config/config.yaml

```diff
@@ -6,6 +6,7 @@ service:
     PORT: 8080
```

Comment:
This is not a fault, it's a request to change the port to something other than 8080. I'm trying to get our docker-compose back up and running, and ras-frontstage is using 8080. Docker will not be an issue as they are mapped behind the routing function. But to aid with developing locally this would be great.

Reply:
PORT can be overridden with an env var, thus would it be an option to do this in the Dockerfile, and avoid any sensitivity to the config file getting changed over time?